router/router.go

```diff
@@ -193,3 +194,28 @@ func Swap(r Router, backend1, backend2 string) error {
 	}
 	return swapBackendName(backend1, backend2)
 }
+
```

Comment:
had to change the struct name to PlanRouter because of the interface Router. is there a better way to handle this?